calcom/cal.com
# imageSrc field is commented for solving this issue and can be removed

## What does this PR do?

Removing the `imageSrc` field mentioned in #9012 

Fixes #9012 



**Environment**: Staging(main branch) / Production

## Type of change



- Commented the field `imageSrc`

## How should this be tested?
1. Create an app with cli yarn app-store create-template
2. Fill in the details.
3. Check config.json file.

![image](https://github.com/calcom/cal.com/assets/66687621/d91092f8-ca76-4da8-9d8f-b91263116b29)
